backend/src/main/java/com/huellapositiva/domain/actions/CancelProposalAction.java

```diff
@@ -0,0 +1,20 @@
+package com.huellapositiva.domain.actions;
+
+import com.huellapositiva.domain.repository.ProposalRepository;
+import com.huellapositiva.infrastructure.orm.entities.JpaProposalStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CancelProposalAction {
+
+    @Autowired
+    ProposalRepository proposalRepository;
+
+    public void executeByReviser(String id){
+        JpaProposalStatus jpaProposalStatus = JpaProposalStatus.builder()
+                .id(6)
```

Comment:
What does 6 means????? :-)
This is an example of magic number. A developer reading this code won't be able to know the meaning of the id unless he/she navigates to the ProposalStatus class.
You can set a constant in the CancelProposalAction class but even better use the ProposalStatus.CANCELLED

Reply:
Yes, that's right. We have just changed it. Thanks!

Reply:
did you push the changes? I can't see them :-)